src/components/Accordion/pageObject/index.js

```diff
@@ -30,6 +30,29 @@ class PageAccordion {
         }
         return null;
     }
+
+    /**
+     * Clicks in the button of the first AccordionSection element.
+     * @method
+     */
+    clickFirstButton() {
+        $(this.rootElement)
+            .$('li:nth-child(1) button')
+            .click();
+    }
+
+    /**
+     * Returns the label of the open AccordionSection
+     * @method
+     * @returns {string}
+     */
+    getOpenSectionLabel() {
+        return $(this.rootElement)
+            .$('div[aria-hidden="false"]')
+            .$('..')
+            .$('div h3 > span:nth-child(2)')
```

Comment:
This is a fragile selector since we can change the markup and then break it. Here you can set:
```data-expanded={isExpanded}```
in the ```StyledHeading ``` component and then select it as:
```
$(this.rootElement)
            .$('[data-expanded="true"]')
            .$('[title="Accordion Label"]')
```
also in this way is more readable